downstream/modules/hub/con-offline-token-active.adoc

```diff
@@ -3,14 +3,14 @@
 
 = Keeping your offline token active
 
-Keeping an offline token active is useful when an application needs to perform action on behalf of the user, even when the user is offline. For example, a routine data backup.
+Keeping an offline token active is useful when an application needs to perform actions on behalf of the user, even when the user is offline. For example, a routine data backup.
```

Comment:
The second sentence here is incomplete. How about: 
Keeping an offline token active is useful when an application needs to perform an action on behalf of the user, like a routine data backup, even when the user is offline.